pages/common/borg.md

```diff
@@ -0,0 +1,32 @@
+# borg
+
+> Deduplicating backup tool with compression and encryption.
+> Supports local and remote backups that are mountable as filesystems.
```

Comment:
I don't see an example which works on remote backups. Perhaps you can switch one of the examples to work on a remote backup.